src/planwise/engine/raster.clj

```diff
@@ -65,6 +65,23 @@
     (throw (ex-info "Failed to open raster file"
                     {:filename path}))))
 
+(defn create-raster
+  [dataset]
+  (let [projection   (.GetProjection dataset)
+        geotransform (.GetGeoTransform dataset)
+        raster-count (.GetRasterCount dataset)
+        band      (.GetRasterBand dataset 1)
+        data-type (.GetRasterDataType band)
+        xsize     (.GetXSize band)
+        ysize     (.GetYSize band)
+        nodata    (read-nodata-value band)
+        data      (read-band-data band 0 0 xsize ysize)]
+    (->Raster projection geotransform xsize ysize data-type nodata data)))
+
+(defn update-raster
```

Comment:
Naming is a bit confusing. Probably move this into a new variant of `create-raster`, since this really is creating a new raster.